resources/user-agents/browsers/mobile-safari/mobile-safari-14-0.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Mobile Safari #MAJORVER#.#MINORVER#",
-  "versions": [14],
+  "versions": ["14.0", "14.1", "14.2", "14.3", "14.4", "14.5", "14.6", 14],
```

Comment:
Please change the order of versions. Please write the highest first. The version "14.0" is not needed, as it the same as version "14". Also change the file name to `mobile-safari-14-x.json`.